### PR TITLE
feat(deploy-to-jexia): enable deploy of applications

### PR DESCRIPTION
**Pull request by marlosin**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [X] The commit message follows [our conventions](../CONTRIBUTING.md#commit-message-format)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [X] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

```
[ ] Bugfix
[X] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?
Currently for deploying to jexia, the parameters have incorrect camelCase naming and env variables aren't required anymore.

## What is the new behavior?
Parameters have correct snake_case naming and no env vars are sent

## Does this PR introduce a breaking change?

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: src/routes/show.svelte [hunk 1]
```diff
@@ -84,23 +84,17 @@
...
     repo_url: repoUrl,
-    env_variables: envVariables,
   }) => {
     if (repoBranch && repoUrl) {
       const queryParam = {
-        repoUrl,
-        repoBranch,
-        env_vars: envVariables
-          ? toQueryString(envVariables, ',')
-          : null,
```
Comment (reinos) [inline src/routes/show.svelte:95]: Is it correct we remove the `env_vars`?

**Comment by reinos** [comment]
Also, keep in mind that we need to set the envVar `DEPLOYMENT_HOST` to the correct url.

File: src/routes/show.svelte (hunk 1 above)
Comment (marlosin) [reply src/routes/show.svelte:95]: yes, as `env_vars` are part of the spec they're not being saved in the datasets

Comment (reinos) [reply src/routes/show.svelte:95]: uhm, no I mean we need to send the `env_vars` somehow to the deployment url right? Now we are not sending any `env_var` to the deployment url.

**Pull request merged**